.jenkins/caffe2/test.sh

```diff
@@ -176,7 +176,7 @@ fi
 ##############
 if [[ "$BUILD_ENVIRONMENT" == *onnx* ]]; then
   pip install -q --user --no-use-pep517 "git+https://github.com/pytorch/vision.git@$(cat .github/ci_commit_pins/vision.txt)"
-  pip install -q --user ninja flatbuffers==2.0 numpy==1.21.5 onnxruntime==1.12.1 beartype==0.10.4
+  pip install -q --user ninja flatbuffers==2.0 numpy==1.21.5 onnxruntime==1.12.1 beartype==0.10.4 onnx==1.12.0
   # numba requires numpy <= 1.20, onnxruntime requires numpy >= 1.21.
   # We don't actually need it for our tests, but it's imported if it's present, so uninstall.
   pip uninstall -q --yes numba
```

Comment:
That does not seem necessary anymore.

Reply:
I think it still needs it cause https://github.com/pytorch/pytorch/blob/954b7ac73b7251e6c62050e5bc3869ff05b1fe95/torch/onnx/utils.py#L1685-L1694

Reply:
I was thinking about the comment about numba. But it is unrelated to your changes.